top-app-bar/Flutter/Flutter-top-app-bar.md

```diff
@@ -148,22 +154,26 @@ When a top app bar transforms into a contextual action bar, the following change
 `AppBar`
 * [Class definition](https://api.flutter.dev/flutter/material/AppBar-class.html)
 * [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/app_bar.dart)
+* [Dartpad Demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/top-app-bar/contextual&gh_ref=develop)
 
 The following example shows a contextual action bar with a contextual title, a close icon, two contextual action icons, and an overflow menu:
 
-![insert screenshot here]()
-
-```
-The source code for the contextual action bar should switch from the regular app bar example to the 
-contextual action bar and include the following:
-* A black contextual container color
-* A close button icon
-* A contextual title that reads "Contextual title" in white
-* Two action items:
-    * download icon
-    * garbage icon
-* Overflow menu
-* The top app bar should remain in place while the user scrolls
+![assets/contextual_screenshot.png](assets/contextual_screenshot.png)
```

Comment:
Replace alt text with "Contextual top app bar with close button, download, garbage, and overflow menu icons"

Reply:
Done